exporter/logicmonitorexporter/config.go

```diff
@@ -0,0 +1,52 @@
+// Copyright  The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logicmonitorexporter // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/logicmonitorexporter"
+
+import (
+	"fmt"
+	"net/url"
+
+	"go.opentelemetry.io/collector/config"
+	"go.opentelemetry.io/collector/config/confighttp"
+	"go.opentelemetry.io/collector/exporter/exporterhelper"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/resourcetotelemetry"
+)
+
+// Config defines configuration for Logic Monitor exporter.
+type Config struct {
+	config.ExporterSettings `mapstructure:",squash"`
+
+	confighttp.HTTPClientSettings `mapstructure:",squash"`
+
+	exporterhelper.QueueSettings `mapstructure:"sending_queue"`
+	exporterhelper.RetrySettings `mapstructure:"retry_on_failure"`
+	ResourceToTelemetrySettings  resourcetotelemetry.Settings `mapstructure:"resource_to_telemetry_conversion"`
+
+	// ApiToken of LM Platform
+	APIToken map[string]string `mapstructure:"apitoken"`
```

Comment:
are only `access_id` and `access_key` possible values? If yes, create a struct with these 2 values instead of `map[string]string`.

Reply:
Yes these are only possible values. Addressed the requested changes.